src/map/modifier.h

```diff
@@ -965,31 +1007,24 @@ enum class Mod
     CRITHITRATE_SLOT       = 1168, // CRITHITRATE for slot
     ATT_SLOT               = 1169, // ATT for slot
     UDMG                   = 1170, // Uncapped dmg taken (all types)
-    SLEEP_MEVA             = 1171, // Sleep MEVA from Barspells
-    POISON_MEVA            = 1172, // Poison MEVA from Barspells
-    PARALYZE_MEVA          = 1173, // Paralyze MEVA from Barspells
-    BLIND_MEVA             = 1174, // Blind MEVA from Barspells
-    SILENCE_MEVA           = 1175, // Silence MEVA from Barspells
-    VIRUS_MEVA             = 1176, // Virus MEVA from Barspells
-    PETRIFY_MEVA           = 1177, // Petrify MEVA from Barspells
 
     // Per PR comment - New ASB section starting at 2000
     TANDEM_STRIKE = 2000, // Beastmaster trait - provides acc/macc to master and pet when both engage the same target
     TANDEM_BLOW   = 2001, // Beastmaster trait - provides subtle blow to master and pet when both engage the same target
 
-    // IF YOU ADD ANY NEW MODIFIER HERE, ADD IT IN scripts/globals/status.lua ASWELL!
+    // IF YOU ADD ANY NEW MODIFIER HERE, ADD IT IN scripts/enum/mod.lua ASWELL!
 
     // The spares take care of finding the next ID to use so long as we don't forget to list IDs that have been freed up by refactoring.
     // 570 through 825 used by WS DMG mods these are not spares.
     //
     // SPARE IDs:
-    // 74 to 79
+    // 75 to 79
     // 138 to 143
     // 156 to 159
-    // 192 to 223
-    // 261 to 280
+    // 217 to 223
+    // 271 to 280
     //
-    // SPARE = 1181, and onward
+    // SPARE = 1076 and onward
```

Comment:
I see you have quite a few modifiers after this one.
Will need to update this number to avoid people duplicating them on accident

Reply:
Thank you :D